From 0374e91c90ee74d4419ac91fb2030d96ee16d544 Mon Sep 17 00:00:00 2001
From: Andreas Herrmann <andreas.herrmann3@amd.com>
Date: Fri, 28 May 2010 13:59:52 +0200
Subject: [PATCH 15/16] Staging: rtl81xx: Fix build problems when CONFIG_CRYPTO=n

Select CONFIG_CRYPTO for all rtl81xx wireless drivers

... to avoid build problems like:

ERROR: "crypto_destroy_tfm" [drivers/staging/rtl8187se/r8187se.ko] undefined!
ERROR: "crypto_alloc_base" [drivers/staging/rtl8187se/r8187se.ko] undefined!
ERROR: "crypto_destroy_tfm" [drivers/staging/rtl8192u/r8192u_usb.ko] undefined!
ERROR: "crypto_alloc_base" [drivers/staging/rtl8192u/r8192u_usb.ko] undefined!
ERROR: "crypto_destroy_tfm" [drivers/staging/rtl8192su/r8192s_usb.ko] undefined!
ERROR: "crypto_alloc_base" [drivers/staging/rtl8192su/r8192s_usb.ko] undefined!
ERROR: "crypto_destroy_tfm" [drivers/staging/rtl8192e/r8192e_pci.ko] undefined!
ERROR: "crypto_alloc_base" [drivers/staging/rtl8192e/r8192e_pci.ko] undefined!

when drivers are built as modules but CONFIG_CRYPTO=n.

Signed-off-by: Andreas Herrmann <andreas.herrmann3@amd.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rtl8192u/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/drivers/staging/rtl8192u/Kconfig
+++ b/drivers/staging/rtl8192u/Kconfig
@@ -3,5 +3,6 @@ config RTL8192U
 	depends on PCI && WLAN && USB
 	select WIRELESS_EXT
 	select WEXT_PRIV
+	select CRYPTO
 	default N
 	---help---
